rust/otap-dataflow/crates/otlp/src/otlp_batch_processor.rs

```diff
@@ -390,6 +371,10 @@ impl GenericBatcher {
     #[must_use]
     pub fn new(config: BatchConfig) -> Self {
         let now = Instant::now();
+        // ToDo: Validate should check self.config.send_batch_size > 0
+        if config.send_batch_size == 0 {
+            panic!("max_batch_size must be greater than zero");
+        }
```

Comment:
The code here had been constructing NodeId values somewhat arbitrarily, and I'm not sure how we're meant to apply validation like this.